src/components/List/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import {map} from 'lodash'
+
+export default ({items}) => (
+  <div>
+    {map(items, (item, index) => (
+      <div
+        key={index}
+        className={`pa4 ${index < items.length - 1 ? 'bb b--light-gray' : ''}`}
+      >
+        {item}
+      </div>
+    ))}
+  </div>
+)
```

Comment:
This adds "cells" of items in a list; generally used inside a widget so it adds padding and a bottom border to all items but the last:

![image](https://cloud.githubusercontent.com/assets/5497885/23816607/03e6bc60-05ab-11e7-901a-e18318439584.png)